core/config/v2/docs/core.toml

```diff
@@ -183,6 +185,41 @@ StartTimeout = '15s' # Default
 # ListenIP specifies the IP to bind the HTTP server to
 ListenIP = '0.0.0.0' # Default
 
+[WebServer.LDAP]
+# Optional LDAP config if WebServer.AuthenticationMethod is set to 'ldap'
+# ServerTls defines the option to require the secure ldaps
+ServerTls = true # Default
+# SessionTimeout determines the amount of idle time to elapse before session cookies expire. This signs out GUI users from their sessions.
+SessionTimeout = '15m0s' # Default
+# QueryTimeout defines how long queries should wait before timing out, defined in seconds
+QueryTimeout = '2m0s' # Default
+# BaseUserAttr defines the base attribute used to populate LDAP queries such as "uid=$", default is example
+BaseUserAttr = 'uid' # Default
+# BaseDn defines the base dn search filter to apply to every LDAP query, default is example
+BaseDn = 'dc=custom,dc=example,dc=com'
+# UsersDn defines the dn base of queries that need to return 'users' structured attributes 
+UsersDn = 'ou=users' # Default
+# GroupsDn defines the dn base of queries that need to return 'groups' structured attributes 
+GroupsDn = 'ou=groups' # Default
+# ActiveAttribute is an optional user field to check truthiness for if a user is valid/active 
+ActiveAttribute = 'organizationalStatus' # Default
+# ActiveAttributeAllowedValue is the value to check against for the above optional user attribute
+ActiveAttributeAllowedValue = 'ACTIVE' # Default
+# AdminUserGroupCn is the name of the LDAP group that maps the core node's 'Admin' role
+AdminUserGroupCn = 'NodeAdmins' # Default
+# EditUserGroupCn is the name of the LDAP group that maps the core node's 'Edit' role
+EditUserGroupCn = 'NodeEditors' # Default
+# RunUserGroupCn is the name of the LDAP group that maps the core node's 'Run' role
+RunUserGroupCn = 'NodeRunners' # Default
+# ReadUserGroupCn is the name of the LDAP group that maps the core node's 'Read' role
+ReadUserGroupCn = 'NodeReadOnly' # Default
```

Comment:
Is `Cn` for `core node`? Would it make sense to group these in their own section:
```suggestion
[WebServer.LDAP.CoreNode]
# AdminUserGroup is the name of the LDAP group that maps the core node's 'Admin' role
AdminUserGroup = 'NodeAdmins' # Default
# EditUserGroup is the name of the LDAP group that maps the core node's 'Edit' role
EditUserGroup = 'NodeEditors' # Default
# RunUserGroup is the name of the LDAP group that maps the core node's 'Run' role
RunUserGroup = 'NodeRunners' # Default
# ReadUserGroup is the name of the LDAP group that maps the core node's 'Read' role
ReadUserGroup = 'NodeReadOnly' # Default
```
Same for `Dn`? And `UserApiToken`?

Reply:
Cn is for Common name, - Refer to https://learn.microsoft.com/en-us/previous-versions/windows/desktop/ldap/distinguished-names

Reply:
Updated comments in docs toml here to specific LDAP 'cn' better